vscode-extension/tsconfig.json

```diff
@@ -18,5 +16,10 @@
     "tsBuildInfoFile": ".tsbuildinfo"
   },
   "include": ["src/**/*.ts"],
-  "exclude": []
+  "exclude": [],
+  "references": [
+    {
+      "path": "../"
```

Comment:
Moved the server into wireit proper. This simplifies the build a lot, though it is a little weird that there's this unused file shipped with wireit.

We talked about maybe shipping the bundled language server, but that would be a separate decision than this, which just moves the source file in so that both typescript and esbuild are happy.

Reply:
Sounds fine, mind filing a issue?

Reply:
+1, filed as https://github.com/google/wireit/issues/210